tavla/app/(admin)/organizations/components/Footer/index.tsx

```diff
@@ -3,22 +3,23 @@ import { useToast } from '@entur/alert'
 import { Heading2, Paragraph } from '@entur/typography'
 import { SubmitButton } from 'components/Form/SubmitButton'
 import { TOrganizationID } from 'types/settings'
-import { setFooter } from './actions'
+import { setFooter as setFooterAction } from './actions'
 import ClientOnlyTextField from 'app/components/NoSSR/TextField'
+import { fireToastFeedback } from 'app/(admin)/utils'
 
 function Footer({ oid, footer }: { oid?: TOrganizationID; footer?: string }) {
     const { addToast } = useToast()
-    return (
-        <form
-            className="box flex flex-col gap-1 "
-            action={async (data: FormData) => {
-                if (!oid) return
-                const info = data.get('footer') as string
 
-                await setFooter(oid, info)
-                addToast('Infomelding lagret!')
-            }}
-        >
+    const setFooter = async (data: FormData) => {
+        if (!oid) return
+        const message = data.get('footer') as string
```

Comment:
Burde vi ikke egentlig sjekke disse to i `setFooterAction`? Det er vel det vi gjør andre plasser pr def

Reply:
Jeg liker at vi sjekker det i action som du sier, clean. Kan endre disse for å bli konsekvent